CODEOWNERS

```
@@ -26,6 +26,7 @@ k8s/**/cluster-*-overlay/ @hmcts/platform-engineering
 k8s/**/common/rpe/namespace.yaml @hmcts/platform-engineering
 k8s/**/common/rpe/identity.yaml @hmcts/platform-engineering
 k8s/**/common/rpe/draft-store-service.yaml @hmcts/platform-engineering
+k8s/**/common/rpe/defaults-limits.yaml @hmcts/platform-engineering
```

default-limits maybe?

it doesn't read very well to me
```suggestion
k8s/**/common/rpe/default-limits.yaml @hmcts/platform-engineering
```